i18n/locales/en.ts

```diff
@@ -11,6 +11,7 @@ const translations = {
     coming_soon: 'Coming soon',
     start_chapter: 'Start chapter',
     back: 'Back',
+    poweroff: 'back to chapter selection page',
```

Comment:
Small nit:

Capitalizing the sentence will look better :)

Also, I think we can remove "page" from the phrase to make it concise.

```suggestion
    poweroff: 'Back to chapter selection',
```